src/react/components/pages/editorPage/editorTagsInput.test.tsx

```diff
@@ -2,26 +2,49 @@ import { mount } from "enzyme";
 import React from "react";
 import MockFactory from "../../../../common/mockFactory";
 import { KeyCodes } from "../../../../common/utils";
-import EditorTagsInput, { IEditorTagsInputProps } from "./editorTagsInput";
+import EditorTagsInput from "./editorTagsInput";
+import { ITagsInputProps } from "vott-react";
 
 // tslint:disable-next-line:no-var-requires
-const TagColors = require("../../common/tagsInput/tagColors.json");
+const TagColors = require("vott-react/dist/lib/components/common/tagColors");
+const ColorCodes = [];
+
+for (const key in TagColors) {
+    if (key) {
+        for (const color in TagColors[key]) {
+            if (color) {
+                ColorCodes.push(TagColors[key][color]);
+            }
+        }
+    }
+}
 
 describe("Tags Input Component", () => {
 
     const originalTags = MockFactory.createTestTags(15);
 
-    function createComponent(props: IEditorTagsInputProps) {
+    function createComponent(props: ITagsInputProps) {
         return mount(
-            <EditorTagsInput {...props}/>,
+            // Listing props one by one because of 'ref' typescript error.
+            // Example: https://github.com/ant-design/ant-design/issues/10405
+            <EditorTagsInput
+                tags={props.tags}
+                onChange={props.onChange}
+                placeHolder={props.placeHolder}
+                delimiters={props.delimiters}
+                onTagClick={props.onTagClick}
+                onCtrlTagClick={props.onCtrlTagClick}
+                onShiftTagClick={props.onShiftTagClick}
+                onCtrlShiftTagClick={props.onCtrlShiftTagClick}
+            />,
         );
     }
 
     it("tags are initialized correctly", () => {
         const onChangeHandler = jest.fn();
         const wrapper = createComponent({
             tags: originalTags,
-            displayHotKeys: true,
+            // displayHotKeys: true,
```

Comment:
Feel free to remove the commented-out prop as well :)

Reply:
i thought i'd leave it to possibly add back in. I think it would be good to be able to witch on and off.

Reply:
I agree, we might add that back in. But I think it'll be cleaner if we just update that when we actually add the property back

Reply:
removed comments